checking kernel supports perf_events... no
Error: Your kernel does not appear to have the necessary support to run oprofile.
configure: error: Unable to build oprofile. Exiting.

---

--- a/configure.ac
+++ b/configure.ac
@@ -115,6 +115,7 @@ if test "$kernel_may_have_perf_events_su
 		rm -f test-for-PERF_EVENT_OPEN
 		AC_LANG_CONFTEST(
 			[AC_LANG_PROGRAM([[#include <linux/perf_event.h>
+				#include <unistd.h>
 				#include <asm/unistd.h>
 				#include <sys/types.h>
 				#include <string.h>
